src/nplinker/metabolomics/GNPSMolecularFamilyLoader.py

```diff
@@ -0,0 +1,83 @@
+import csv
+from typing import Tuple, List
+from nplinker.logconfig import LogConfig
+from nplinker.metabolomics.IMolecularFamilyLoader import IMolecularFamilyLoader
+from nplinker.metabolomics.molecular_family import MolecularFamily
+from nplinker.metabolomics.singleton_family import SingletonFamily
+
+
+logger = LogConfig.getLogger(__file__)
+
+
+class GNPSMolecularFamilyLoader(IMolecularFamilyLoader):
+    def __init__(self, filename: str):
+        self._families: List[MolecularFamily] = []
+        
+        for family_id, spectra_ids in _load_molecular_families(filename).items():
+            if family_id == -1:
+                for spectrum_id in spectra_ids:
+                    family = SingletonFamily()
+                    family.spectra_ids = [spectrum_id]
```

Comment:
`[spectrum_id]` should be the "set" `spectrum_id`

Reply:
fixed in 6298a57: Changed spectra_ids variable type to set[int]